release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -311,6 +311,21 @@ With this release, the etcd tuning parameters can be set to values that optimize
 * `Standard`
 * `Slower`
 
+[discrete]
+[id="ocp-4-16-unauthenticated-users-cluster-role-bindings_{context}"]
+=== Unauthenticated users no longer have access to cluster role bindings
+
+In {product-title} {product-version} unauthenticated users will no longer have default access to cluster role bindings. This change is for new clusters, previous clusters will continue to allow unauthenticated access to cluster role bindings.
```

Comment:
#### Affects only a 4 ClusterRoleBindings

 would narrow it down:

"to specific cluster role bindings that are necessary for certain feature sets".

We should avoid to give the impression, that unauthenticated have access to a broad range of cluster roles.

#### Role vs Role Binding

"Access to a role" or "Being part of a role binding".